extraction/pom.xml

```diff
@@ -71,6 +71,27 @@
       <artifactId>parquet-hadoop</artifactId>
       <version>1.6.0rc1</version>
     </dependency>
+    
+    <!-- Flink dependencies -->
```

Comment:
please remove those, we will handle the flink related stuff in the analysis module